rest-catalog/quarkus-rest-catalog/build.gradle.kts

```diff
@@ -59,10 +59,8 @@ dependencies {
 
   implementation(enforcedPlatform(libs.quarkus.bom))
   implementation(enforcedPlatform(libs.quarkus.amazon.services.bom))
-  implementation("org.jboss.resteasy:resteasy-core-spi")
-  implementation("io.quarkus:quarkus-resteasy")
-  implementation("io.quarkus:quarkus-resteasy-jackson")
-  implementation("io.quarkus:quarkus-reactive-routes")
+  implementation("io.quarkus:quarkus-resteasy-reactive")
```

Comment:
As stated elsewhere the switch to resteasy-reactive allows to inject request context anywhere, not only on resource classes. So far I haven't seen any inconvenience or regression, apart maybe from the fact that the Iceberg ObjectMapper bean now must be explicitly produced.